J/Julia/build_tarballs.jl

```diff
@@ -72,32 +72,23 @@ make ${BUILD_FLAGS[@]} install
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = [
-    Linux(:x86_64; libc=:glibc, compiler_abi=CompilerABI(:gcc7)),
-    Linux(:i686; libc=:glibc, compiler_abi=CompilerABI(:gcc7)),
-    Windows(:x86_64; compiler_abi=CompilerABI(:gcc7)),
-    Windows(:i686; compiler_abi=CompilerABI(:gcc7)),
+    Linux(:x86_64; libc=:glibc),
+    Linux(:i686; libc=:glibc),
+    Windows(:x86_64;),
+    Windows(:i686;),
 ]
```

Comment:
Why not ` platforms = supported_platforms()`?

Reply:
We don't build Julia for all the supported platforms upstream (such as musl).